policyengine_us/parameters/gov/states/mt/dhs/tanf/income/deductions/earned_income_disregard/percentage.yaml

```diff
@@ -0,0 +1,12 @@
+description: Montana excludes this share of earnings from the Temporary Assistance for Needy Families program countable income, when computing the benefit value.
+
+values:
+  2018-01-01: 0.25
+
+metadata:
+  unit: /1
+  period: month
+  label: Montana TANF earned income percentage disregard
+  reference:
+    - title: TANF 602-1 Income Disregards
+      href: https://dphhs.mt.gov/assets/hcsd/tanfmanual/tanf602-1jan012018.pdf
```

Comment:
Could you also add the [legal code](https://www.law.cornell.edu/regulations/montana/Mont-Admin-r-37.78.406) to the references in the parameter files